src/components/Nav/LogIn/Kakao.js

```diff
@@ -0,0 +1,74 @@
+import { useEffect } from 'react';
+import qs from 'qs';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import variables from '../../../styles/variables';
```

Comment:
```suggestion
import { useEffect } from 'react';
import { useLocation } from 'react-router-dom';
import styled from 'styled-components';
import qs from 'qs';
import variables from '../../../styles/variables';
```
[리팩토링 가이드](https://study.wecode.co.kr/session/content/56)에 9번 을 참고 해주세요~